bin/reth/src/node/mod.rs

```diff
@@ -285,6 +294,16 @@ impl Command {
     }
 }
 
+/// Dumps peers to `file_path` for persistence.
+async fn dump_peers(file_path: &Path, network: NetworkHandle) -> Result<(), io::Error> {
+    info!(target : "net::peers", file = %file_path.display(), "Saving current peers");
+    let writer = std::io::BufWriter::new(std::fs::File::create(file_path)?);
+    let known_peers = network.peers_handle().all_peers().await;
```

Comment:
since this is time sensitive, let's use the `tokio::fs::File::create` variant so we can create the file and fetch the peers concurrently via `select`

Reply:
but perhaps this is not even required and we can use `fs::write` directly?